### ISSUE TITLE
MMS Needs to be updated to allow array of media (not Catapult URL)

### ISSUE THREAD

**Issue by dtolb** [bandwidthcom/php-bandwidth#15]
MMS should allow an array of URLs to send as one payload.

MMS no longer requires the URL to be a valid catapult url, any URL can be used.

http://ap.bandwidth.com/docs/rest-api/messages/

New Messages should be created like:

```PHP
$mms_array = array('<url1>', '<url2>', '<url3');

$message1 = new Catapult\Message(array(
    "from" => $argv[1],
    "to" => $argv[2],
    "text" => $text1,
    "media" => $mms_array
));
```

**Comment by dtolb** [comment]
@ffranco-daitan @guaycuru (which name to use, I forget) Is this something you can do !? :)

**Comment by ffranco-daitan** [comment]
@dtolb Yes, I think I can do it, let's ask @EdHintz about priorities / when should this be done. (btw Guaycuru is my personal account, ffranco-daitan is my Daitan account)
